plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/transactionlog/DeltaLakeParquetStatisticsUtils.java

```diff
@@ -127,6 +127,20 @@ public static Object jsonValueToTrinoValue(Type type, @Nullable Object jsonValue
             return (long) floatToRawIntBits((float) (double) jsonValue);
         }
         if (type == DOUBLE) {
+            if (jsonValue instanceof String) {
+                switch ((String) jsonValue) {
+                    case "Infinity" -> {
+                        return Double.POSITIVE_INFINITY;
+                    }
+                    case "-Infinity" -> {
+                        return Double.NEGATIVE_INFINITY;
+                    }
+                    case "NaN" -> {
```

Comment:
can it be `nan` or `NAN`?

Reply:
I do not believe so, from what I've seen this is how Trino writes the value in a delta lake entry.